play-json/src/main/scala/org/http4s/play/instances/package.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package play
+
+package object instances {
+  object all extends AllInstances
+  object play extends PlayInstances
```

Comment:
In scala-xml, we called these `ElemInstances`, because they were for `Elem`.  Following that pattern, we could perhaps split this into `JsValue`, `Reads`, and `Writes`?  I'm not sure whether people will ever want that level of granularity, but it seems consistent with Cats.

If we don't split it up, I think we might just stick with `AllInstances` and `all`.